src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/DataTestUtility.cs

```diff
@@ -646,13 +646,15 @@ public static string RetrieveValueFromConnStr(string connStr, string[] keywords)
         public class TraceEventListener : EventListener
         {
             public List<int> IDs = new List<int>();
+            public EventSource EventSourceProperty;
```

Comment:
Do you need this here?

Reply:
It was added for more event source testing. I will remove it for now to investigate the failure of the pipelines.